contracts.go

```diff
@@ -224,3 +220,68 @@ func compileContract(cmd *cobra.Command, args []string) error {
 
 	return nil
 }
+
+func reviseContract(cmd *cobra.Command, args []string) error {
+
+	// read the params file
+	params_b, err := ioutil.ReadFile("params.toml")
+	if err != nil {
+		return err
+	}
+
+	// read the contract template
+	template_b, err := ioutil.ReadFile("template.md")
+	if err != nil {
+		return err
+	}
+
+	// open history file (or create history file if it doesn't exist)
+	file, err := os.OpenFile("history.toml", os.O_RDWR|os.O_CREATE, 0600)
+	if err != nil {
+		return err
+	}
+
+	defer file.Close()
+
+	if _, err = file.WriteString("// Do not modify this file yourself under any circumstances!"); err != nil {
+		return err
+	}
+
+	// read history file
+	history_b, err := ioutil.ReadFile("history.toml")
+	if err != nil {
+		return err
+	}
+
+	// combine params, history, and template data
+	all_b := [3][]byte{params_b, template_b, history_b}
+	byte_array := make([]byte, 3, 3)
+
+	for _, element := range all_b {
+		for _, b := range element {
```

Comment:
`cannot use element (type []byte) as type byte in append`

each `element` is `type []byte` whereas each `b` is `type byte`